apps/api/src/scraper/WebScraper/single_url.ts

```diff
@@ -447,7 +447,7 @@ export async function scrapSingleUrl(
     let linksOnPage: string[] | undefined;
 
     if (pageOptions.includeLinks) {
-      linksOnPage = extractLinks(rawHtml, urlToScrap);
+      linksOnPage = extractLinks(html, urlToScrap);
```

Comment:
I think the problem with this is that rely on this function for our /crawl, which will end up failling to grab all the links if we don't pass the raw version

Reply:
@mogery lmk if im wrong

Reply:
A global code search did not reveal any usage of the `linksOnPage` field anywhere other than api return
I believe the crawler uses a separate extractLinksFromHTML function
https://github.com/mendableai/firecrawl/blob/8a4f4cb9d98884bc70f4cf188a2c4dc87f656462/apps/api/src/services/queue-worker.ts#L374 and 
https://github.com/mendableai/firecrawl/blob/8a4f4cb9d98884bc70f4cf188a2c4dc87f656462/apps/api/src/scraper/WebScraper/crawler.ts#L382

Reply:
Right makes sense.